packages/core/theme/src/colors/__tests__/semantic.test.ts

```diff
@@ -0,0 +1,36 @@
+import {getContrast} from "color2k";
+import get from "lodash.get";
+
+import {semanticColors} from "../semantic";
+
+const guidelines = {decorative: 1.5, readable: 3, aa: 4.5, aaa: 7};
+const targetGuideline: keyof typeof guidelines = "aa";
```

Comment:
Those are the worst offenders who even fail when `targetGuideline` is set to `readable`.
```
      ✕ light.success.DEFAULT has enough contrast with light.background.DEFAULT to be readable (8 ms)
      ✕ light.success.foreground has enough contrast with light.success.DEFAULT to be readable (1 ms)
      ✕ light.warning.DEFAULT has enough contrast with light.background.DEFAULT to be readable (1 ms)
      ✕ light.warning.foreground has enough contrast with light.warning.DEFAULT to be readable (1 ms)

```